src/google_cloud_debugger/google_cloud_debugger_lib/compiler_helpers.cc

```diff
@@ -311,9 +311,15 @@ CorElementType TypeCompilerHelper::ConvertStringToCorElementType(
   }
 }
 
+struct CorElementTypeHash {
+  std::size_t operator()(CorElementType cor_type) const {
```

Comment:
This hash function just casts the enum `CorElementType` to an unsigned int. Since `CorElementType` is an `enum`, each `CorElementType` is a unique integral so this hash function will give unique hash value for each `CorElementType` value.